test.ts

```diff
@@ -138,3 +181,16 @@ async function randomEvent(ctx: InMemoryAccountContext, kind?: NostrKind, conten
     });
     return event;
 }
+
+async function queryGql(relay: Relay, query: string) {
+    const { hostname, port } = new URL(relay.url);
+    const res = await fetch(`http://${hostname}:${port}/api`, {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+            "password": "123",
```

Comment:
please pass in the password or expose it from the `relay` object.

Reply:
ok